kagami/server.py

```diff
@@ -17,6 +23,13 @@ async def lifespan(app: FastAPI):
     global supervisor
     supervisor = Supervisor  # TODO load the supervisor
 
+    logger.info("Reading config from config.env")
+    logger.info(f"gRPC Host: {config.grpc_host}")
+    logger.info(f"gRPC Port: {config.grpc_port}")
+    logger.info(f"Log File: {config.log_file}")
+    logger.info(f"Log Level: {config.log_level}")
+    logger.info(f"Database URL: {config.database_url}")
```

Comment:
[Database URL](https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls) may contain sensitive information such as passwords and may not be suitable for display in logs.

Reply:
That's true, but the configuration file (config.env) already contains the information on the disk. Is there a better solution for this?

Reply:
Separate environment variables might be a solution, here is an example:

config.env  
```
DB_IP=xxx
DB_PORT=xxx
```

secret.env  
```
DB_PASSWORD=xxx
DB_USER=xxx
```

or use key management tools like HashiCorp Vault or AWS Secrets Manager, but we need to consider ROI at this stage. This may be a direction for future improvement.

Reply:
For other parts, or it's unnecessary to be overly concerned about security, LGTM.

Reply:
> Separate environment variables might be a solution and use ACL to control the read permission, here is an example:
> 
> config.env
> 
> ```
> DB_IP=xxx
> DB_PORT=xxx
> ```
> 
> secret.env
> 
> ```
> DB_PASSWORD=xxx
> DB_USER=xxx
> ```
> 
> or use key management tools like HashiCorp Vault or AWS Secrets Manager, but we need to consider ROI at this stage. This may be a direction for future improvement.

That's a good idea, and i will remove the part of printing configs in logger.